tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/cocosnet_evaluator.py

```diff
@@ -0,0 +1,477 @@
+            print_info('\tshape: {}\n'.format(output_info.shape))
+
+
+class CocosnetModel(BaseModel):
+    def __init__(self, network_info, launcher, delayed_model_loading=False):
+        self.net_type = "cocosnet_network"
+        self.adapter = create_adapter(network_info.get('adapter'))
+        super().__init__(network_info, launcher, delayed_model_loading)
+        self.adapter.output_blob = self.output_blob
+
+    def set_input_and_output(self):
+        has_info = hasattr(self.exec_network, 'input_info')
+        if has_info:
+            inputs_data = OrderedDict([(name, data.input_data) for name, data in self.exec_network.input_info.items()])
+        else:
+            inputs_data = self.exec_network.inputs
+        self.inputs_names = list(inputs_data.keys())
+        self.output_blob = next(iter(self.exec_network.outputs))
+        self.adapter.output_blob = self.output_blob
+
+    def fit_to_input(self, input_data):
+        inputs = {}
+        for value, key in zip(input_data, self.inputs_names):
+            value = np.expand_dims(value, 0)
+            value = np.transpose(value, (0, 3, 1, 2))
+            inputs.update({key: value})
+        return inputs
+
+    def predict(self, identifiers, inputs):
+        results = []
+        for current_input in inputs:
+            prediction = self.exec_network.infer(self.fit_to_input(current_input))
+            results.append(*self.adapter.process(prediction, identifiers, [{}]))
+        return results
+
+
+class GanCheckModel(BaseModel):
+    def __init__(self, network_info, launcher, delayed_model_loading=False):
+        self.net_type = "verification_network"
+        self.additional_layers = network_info.get('additional_layers')
+        super().__init__(network_info, launcher)
+
+    def load_model(self, network_info, launcher, log=False):
+        model, weights = self.auto_model_search(network_info, self.net_type)
+        if weights:
+            self.network = launcher.read_network(model, weights)
+            for layer in self.additional_layers:
+                self.network.add_outputs(layer)
+            self.exec_network = launcher.ie_core.load_network(self.network, launcher.device)
+        else:
+            self.network = None
+            self.exec_network = launcher.ie_core.import_network(str(model))
+        self.set_input_and_output()
+        if log:
+            self.print_input_output_info()
+
+    def set_input_and_output(self):
+        has_info = hasattr(self.exec_network, 'input_info')
+        input_info = self.exec_network.input_info if has_info else self.exec_network.inputs
+        self.input_blob = next(iter(input_info))
+        self.output_blob = list(self.exec_network.outputs.keys())
+        self.number_of_metrics = len(self.output_blob)
+
+    def fit_to_input(self, input_data):
+        input_data = cv2.cvtColor(input_data, cv2.COLOR_RGB2BGR)
+        input_data = cv2.resize(input_data, dsize=(299, 299))
```

Comment:
input data size is model specific parameter, so I think it will be better if you will use height and width from network input instead of hard coded value (majority imagenet trained models has input 224x224)